### PR TITLE
Convert AcquisitionPassthroughGadget to ChannelGadget

### PR DESCRIPTION
**Pull request by Andrew-Dupuis**
Continuing #1087, this PR converts AcquisitionPassthroughGadget to a ChannelGadget using the conventions established in #1090 and #1094.

### CODE REVIEW

**Comment by hansenms** [comment]
@Andrew-Dupuis, you probably want to get up-to-date on master here and there seems to be some other issue.

**Comment by Andrew-Dupuis** [comment]
Yeah - not sure what happened with his one, looked fine on my laptop, but used the GitHub dev container for the push and things look a bit goofed up. I'll request a review once I figure out where the wires got crossed - sorry for that!

File: gadgets/debugging/AcquisitionPassthroughGadget.h [hunk 1]
```diff
@@ -1,24 +1,23 @@
-#ifndef ACQUISITIONPASSTHROUGHGADGET_H
-#define ACQUISITIONPASSTHROUGHGADGET_H
+4 lines (21 sloc) 740 Bytes
```
Comment (Andrew-Dupuis) [inline gadgets/debugging/AcquisitionPassthroughGadget.h:1]: ....this shouldn't have been copy pasted....

**Comment by Andrew-Dupuis** [comment]
Looks like we're all set now - apologies for the messed-up copy paste.

**Review by roopchansinghv — APPROVED** [review]
Looks good - glad you caught that.

**Review by hansenms — APPROVED** [review]

**Pull request merged**